src/stats.rs

```diff
@@ -99,6 +128,27 @@ impl Reporter {
         };
     }
 
+    pub fn register_client(
+        &self,
+        process_id: i32,
+        pool_name: String,
+        username: String,
+        app_name: String,
+    ) {
```

Comment:
This walks around the need to send `app_name`, `username` and `pool_name` to send these information with every client event. 

We can do that anyway to keep the stats interface stateless